tools/sgen/input/FluentUIStyle.yml

```diff
@@ -150,8 +151,9 @@ Opacity:
   none: 1f
   clear: 0.0f
 
-Shadow:
-    shadow28: [ blur: 28pt, color: $Colors.Neutral.black, opacity: $Opacity.opacity24, x: 0pt, y: 14pt ]
+Shadow28:
```

Comment:
I've sent a follow-up question to design on this.
I think we should have all Shadow values under the same Appearance Proxy.

Think of the scenario when we create Shadow32 for example. It will be a different Appearance Proxy instead of another variation of Shadow.

If we get the confirmation that the shadows are always composed of 2 fixed values, we can make those named properties and avoid the [0] and [1] use throughout the code.

Reply:
Updated as per our discussion